### ISSUE TITLE
Python environment issue

### ISSUE THREAD

**Issue by soto-13** [NVlabs/conv-tt-lstm#6]
Hello

Are the requirements.txt provided in both the code/ and codeopt/ up to date? I had the following issues:

1. Inexistent versions for pycurl/pygobject/python-apt/torch/unattended-upgrades (if I remove the version number it advances though, except for python-apt and unattended-upgrades which I had to remove completely)
2. Building wheels for pygobject, PyWavelets, scipy, scikit learn fails too for these versions

In short, does 
```
python3.8 -m venv test
source test/bin/activate
pip3 install -U pip
pip3 install -r code/requirements.txt
```
work for you?

**Comment by wonmin-byeon** [comment]
We used requirements.txt for python 3.6. Thanks for letting us know. I will update our README accordingly. 

Does the code run after removing the versions? 
Installing newer version of other libraries should also work with our code. FYI, python 3.8 with torch 1.8 works fine for me (using docker nvcr.io/nvidia/pytorch:21.02-py3). 
I can recreate requirements.txt for you with the latest version (I'm using) if needed.

**Comment by soto-13** [comment]
I haven't had a built environment to run the code. Sequentially, the following happens, (this is for the requirements.txt within code/) on a linux machine:

1. The pycurl version does not exist - seems to be a minor detail as removing the version gives a very close one. I noticed it is indeed happier with python3.6
2. The pygobject version does not exist - here less trivial: 3.20.0 does not seem to exist anywhere (the website shows it's not there [)](https://pypi.org/project/PyGObject/#history) - I simply removed it and kept running. However it will complain with python3.6 (it attempts to install pygobject 3.38.0) but not with python3.8. I figured you probably meant pygobject 3.30.0 as it runs well with python3.6. Python3.8 is happy with either the latest version or 3.30.0.
3. Both pythons complain with python-apt==1.1.0b1+ubuntu0.16.4.2. Perhaps you were using ubuntu on your machine, I just removed it.
4. Python3.8 will complain about the torch version, I gave it the latest one (python3.6 is happy with the one given)
5. Both pythons complain about unattended-upgrades, I removed it

However after collecting all packages with python3.6, there is a failure when installing pycurl:
`  Running setup.py install for pycurl ... error`

Installing pycurl with python3.8 works, but python3.8 will fail with pygobject using either 3.30.0 or the latest one.

All in all I think it would be best if possible that you provide the 'golden combo' you currently use. It seems like the simplest solution instead of navigating the jungle of pythons and package versions. thanks!

**Comment by soto-13** [comment]
Also: do you happen to have a copy of the .npz files you used for training and testing? it is also a bit problematic in my case to go from the given .gz and .npy to the .npz (ref. ddpae-video-prediction)

**Comment by wonmin-byeon** [comment]
Sorry for the late reply. Let's discuss your specific problems via email. Please send me an emails with your issues of creating the dataset. Also, I can share you requirements. txt for python 3.8.
